README.md

```diff
@@ -4,14 +4,23 @@
 
 Execute
 ```shell
-docker build --no-cache -t eclipse/che-plugin-marketplace .
+docker build --no-cache -t eclipse/che-plugin-registry .
 ```
 Where `--no-cache` is needed to prevent usage of cached layers with plugin registry files.
 Useful when you change plugin metadata files and rebuild the image.
 
 ## https://hub.docker.com/
 
-```eclipse/che-plugin-marketplace:latest``` image would be rebuilt after each commit in master
+```eclipse/che-plugin-registry:latest``` image would be rebuilt after each commit in master
+
+## OpenShift
+You can deploy Che plugin registry on Openshift with command.
+```
+  oc new-app -f openshift/che-plugin-registry.yml \
```

Comment:
As far as I understand there are default values for the specified parameters IMAGE, IMAGE_TAG, PULL_POLICY. Would it better to have the simplest command here like oc new-app -f openshift/che-plugin-registry.yml and separately describe parameters that can be overridden?